server/src/main/java/org/opensearch/index/translog/transfer/TranslogTransferManager.java

```diff
@@ -68,7 +69,7 @@ public TranslogTransferManager(
     ) {
         this.shardId = shardId;
         this.transferService = transferService;
-        this.remoteBaseTransferPath = remoteBaseTransferPath;
+        this.remoteBaseTransferPath = remoteBaseTransferPath.add(DATA_DIR);
         this.remoteMetadataTransferPath = remoteBaseTransferPath.add(METADATA_DIR);
```

Comment:
this looks wrong as we are appending `DATA_DIR` as well in previous line.

Reply:
we appended in instance variable, this is the parameter to constructor

Reply:
oh got it . renaming should clear the confusion now.